### PR TITLE
docs my first app

### PR DESCRIPTION
**Pull request by alexcg1**
My First Jina App - this tutorial explains how to build wikipedia search example for yourself

- docs(my_first_app): initial commit
- docs(my_first_app): move toc
- docs(my_first_app): rm toc
- docs(my_first_app): improve wording and structure
- fix(my-first-app): url slash
- docs(my-first-app): fix all TODOs

### CODE REVIEW

File: chapters/my_first_jina_app/index.md
Comment (FionnD) [inline chapters/my_first_jina_app/index.md]: The file should be in chapters/my_first_jina_app.md

Comment (FionnD) [reply chapters/my_first_jina_app/index.md]: and images added to an image folder in the root

File: chapters/my_first_jina_app/index.md
Comment (bwanglzu) [inline chapters/my_first_jina_app/index.md]: ~maybe you can give a short explanation on `-t` refers to `task`~

File: chapters/my_first_jina_app/index.md
Comment (FionnD) [inline chapters/my_first_jina_app/index.md]: What are key concepts? Things you should have looked at before, or things you learn by reading this?

File: chapters/my_first_jina_app/index.md
Comment (bwanglzu) [inline chapters/my_first_jina_app/index.md]: I guess it's  `python app --task=query`

File: chapters/my_first_jina_app/index.md
Comment (bwanglzu) [inline chapters/my_first_jina_app/index.md]: ah I saw it here, please ignore my last message

File: chapters/my_first_jina_app/index.md
Comment (FionnD) [inline chapters/my_first_jina_app/index.md]: Add a link here maybe? https://docs.jina.ai/chapters/incremental_indexing/index.html

File: chapters/my_first_jina_app/index.md
Comment (FionnD) [inline chapters/my_first_jina_app/index.md]: next

**Review by bwanglzu — APPROVED** [review]
great work!

File: chapters/my_first_jina_app/index.md
Comment (FionnD) [inline chapters/my_first_jina_app/index.md]: It will take a user's input and return a list of Wikipedia sentences that match most closely.

File: chapters/my_first_jina_app/index.md
Comment (FionnD) [inline chapters/my_first_jina_app/index.md]: check it's always front-end and you don't use frontend some times

File: chapters/my_first_jina_app/index.md
Comment (FionnD) [inline chapters/my_first_jina_app/index.md]: In this section

Comment (FionnD) [reply chapters/my_first_jina_app/index.md]: full spot

Comment (FionnD) [reply chapters/my_first_jina_app/index.md]: stop*

File: chapters/my_first_jina_app/index.md
Comment (FionnD) [inline chapters/my_first_jina_app/index.md]: in the future

**Review by FionnD — CHANGES_REQUESTED** [review]
Looks great to me! Really good guide.

File: chapters/my_first_jina_app/index.md
Comment (Yongxuanzhang) [inline chapters/my_first_jina_app/index.md]: This image is not showing.
```suggestion
<img src="https://docs.jina.ai/_images/pod.png width="20%" align="left">
```

**Review by Yongxuanzhang — COMMENTED** [review]
Awesome work! 
I prefer to add a table of content at the beginning.

File: chapters/my_first_jina_app/index.md
Comment (alexcg1) [reply chapters/my_first_jina_app/index.md]: Moved the markdown file. However I don't see any `images` folder in root of master branch. Should I create it myself?

File: chapters/my_first_jina_app/index.md
Comment (alexcg1) [reply chapters/my_first_jina_app/index.md]: Good catch. I moved this to first time we use `-t`

File: chapters/my_first_jina_app/index.md
Comment (alexcg1) [reply chapters/my_first_jina_app/index.md]: For ToC is there an automatic way to do this @bwanglzu @FionnD ? Otherwise it's a major time-sink to create and (more importantly) maintain

Comment (alexcg1) [reply chapters/my_first_jina_app/index.md]: Fixed image

File: chapters/my_first_jina_app/index.md
Comment (FionnD) [reply chapters/my_first_jina_app/index.md]: It's part of an ongoing PR #98 . I suppose you can wait to merge or add it yourself

File: chapters/my_first_jina_app/index.md
Comment (FionnD) [reply chapters/my_first_jina_app/index.md]: https://ecotrust-canada.github.io/markdown-toc/

File: chapters/my_first_jina_app/index.md
Comment (alexcg1) [reply chapters/my_first_jina_app/index.md]: I'll hold off until it's merged

File: chapters/my_first_jina_app.md
Comment (Yongxuanzhang) [inline chapters/my_first_jina_app.md:7]: This gif seems missing?

**Review by Yongxuanzhang — COMMENTED** [review]
Is this section added to the index? Or it will be organized later?

File: chapters/my_first_jina_app/index.md
Comment (FionnD) [reply chapters/my_first_jina_app/index.md]: I have reviewed it but you can hold off

**Review by FionnD — APPROVED** [review]

**Pull request merged**
